package.json

```diff
@@ -85,7 +85,7 @@
     "@types/lodash.merge": "^4.6.7",
     "@types/lodash.uniqueid": "^4.0.7",
     "@types/proj4leaflet": "^1.0.10",
-    "@types/react": "^18.0.33",
+    "@types/react": "^18.2.45",
```

Comment:
I always find it confusing when types packages' version seems to be aligned with the actual package. Is `@types/react@18.2.x` compatible with `react@18.2.x`? I guess so

Reply:
I think and hope so :grimacing: